sdk/communication/azure-communication-rooms/tests.yml

```diff
@@ -4,4 +4,17 @@ stages:
   - template: /sdk/communication/communication-tests-template.yml
     parameters:
       PackageName: azure-communication-rooms
-      SafeName: azurecommunicationrooms
\ No newline at end of file
+      SafeName: azurecommunicationrooms
+      CloudConfig:
+        Public:
+          SubscriptionConfigurations:
+            - $(sub-config-azure-cloud-test-resources)
```

Comment:
just wondering why PPE does not need this sub-config-azure-cloud-test-resources? thanks in advance @mikehang-msft !

Reply:
Hi @allchiang-msft, sub-config-azure-cloud-test-resources config contains a list of resource we use in PROD (Public), those won't be able to use in PPE or INT. For PPE, we are using one configured for PPE containing PPE keyword in its name. In this case, we have two configurations. Hope that helps,

    - $(sub-config-communication-ppe-test-resources-common)
    - $(sub-config-communication-ppe-test-resources-java)